gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Daemon’s heap size
-org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=512m -XX:MaxMetaspaceSize=1536m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
+org.gradle.jvmargs=-Xmx4g -XX:MaxMetaspaceSize=1536m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
```

Comment:
IIRC, `MaxPermSize` was helping with Android builds.

Reply:
It's been removed in Java 17, see https://stackoverflow.com/questions/69402824/java-17-with-maven-wrapper-results-in-unrecognized-vm-option-maxpermsize-512m